### ISSUE TITLE
AppCleaner finds fewer results on v5.3.5

### ISSUE THREAD

**Issue by Imany42** [d4rken/sdmaid-public#5177]
My phone (Poco X3 Pro) got update 5.3.5 today.  I did the update but it no longer works as well as the previous version.  Normally, in the previous version, it would clear the cache of the applications, it would find the leftover files.  Can't find anything now.  It always looks clean. I wish you good work

**Comment by d4rken** [comment]
Can you share a debuglog?
https://github.com/d4rken/sdmaid-public/wiki/Reporting-a-bug#debug-log

**Comment by Imany42** [comment]
Dude I deleted version 5.3.5  but if it will solve your problem I can
reinstall and do whatever you want.  I am currently using version 5.2.2 and
it works fine, no problem.  I also installed version 5.3.4 but had the same
problem.  I installed version 5.2.2 and there is no problem with this
version, it works fine.

I'm sorry, my English is not good.  I use translation, I hope you can
understand.

7 Kas 2021 Pzr 18:11 tarihinde Matthias Urhahn ***@***.***>
şunu yazdı:

**Comment by d4rken** [comment]
Without a debug log I will not be able to investigate this :frowning:

**Comment by Imany42** [comment]
Sorry dude, I'm asking because I don't know.  Will my private information
not appear in the log file you want?

8 Kas 2021 Pzt 10:40 tarihinde Matthias Urhahn ***@***.***>
şunu yazdı:

**Comment by d4rken** [comment]
File names will appear in the log.
You can also mail it to me if you don't want to post it.

**Comment by Imany42** [comment]
I found the problem while changing the app version to save the log file
bro.  Cannot find memory.  In version 5.2.2, after allowing the application
to access the storage, the option 'found a secondary memory' appeared and
showed the phone's memory.  In version 5.3.5 it only asks for storage
permission but does not see memory.

I hope I was able to explain.  Good luck

8 Kas 2021 Pzt 18:48 tarihinde Matthias Urhahn ***@***.***>
şunu yazdı:
